workflow-examples/src/main/java/com/redhat/parodos/examples/move2kube/utils/Move2KubeUtils.java

```diff
@@ -0,0 +1,21 @@
+package com.redhat.parodos.examples.move2kube.utils;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * @author Gloria Ciavarrini (Github: gciavarrini)
+ */
+public abstract class Move2KubeUtils {
+
+	private Move2KubeUtils() {
+	}
+
+	public static String getPath(String server, String workspaceID, String projectID, String outputID)
+			throws URISyntaxException {
+		String path = String.format("/workspaces/%s/projects/%s/outputs/%s", workspaceID, projectID, outputID);
```

Comment:
nit: can you replace with 
`String path = "/workspaces/%s/projects/%s/outputs/%s".formatted(workspaceID, projectID, outputID)`